### DISCUSSION TITLE
Why is username squatting not a reportable option anymore

### DISCUSSION THREAD

**Discussion by PoIygon** [community/community#23913] [New to GitHub]
<p>it is useful if someone name sniped a username and you want to have it. i really wanted to get the username bob</p>

**Comment by tajmone** [comment, marked as answer]
<p>Please see this thread on the topic:</p>
<aside class="quote quote-modified" data-post="1" data-topic="189177">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/inactive-username-cant-be-claimed/189177">Inactive username can't be claimed?</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-account-questions/42"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #fb8532;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Having problems with your GitHub Account? Get Help here.">GitHub Account Questions</span></a>
  </div>
  <blockquote>
    Lately, I’ve been little sensitive about the usernames across the websites I hold an account and started to use the same username everywhere possible. There is this guy who is blocking the username I wanted on github, from over a decade with no public activates and repositories. 
To claim this username I have reached out to github support and requested to release it so that I can claim. But github support rejected my request, stating it can’t be release to me. I still don’t know why I can’t clai…
  </blockquote>
</aside>
